src/python/grpcio/grpc/aio/_base_server.py

```diff
@@ -211,6 +211,22 @@ def invocation_metadata(self) -> Optional[Metadata]:
           The invocation :term:`metadata`.
         """
 
+    @abc.abstractmethod
+    def is_active(self) -> bool:
+        """ Checks whether the context is active
+
+        Returns:
+          boolean conveying whether the context is active
+        """
+
+    @abc.abstractmethod
+    def add_callback(self, callback) -> None:
```

Comment:
This should be called `add_done_callback`: https://grpc.github.io/grpc/python/grpc_asyncio.html#grpc.aio.RpcContext.add_done_callback